clients/vault/src/oracle/collector/collector.rs

```diff
@@ -374,47 +371,44 @@ mod test {
 		let env_map =
 			EnvelopesFileHandler::get_map_from_archives(env_slot).expect("should return a map");
 
-		// let txset_slot = 42867088;
-		// let txsets_map =
-		// 	TxSetsFileHandler::get_map_from_archives(txset_slot).expect("should return a map");
+		let txset_slot = 92910;
+		let txsets_map =
+			TxSetsFileHandler::get_map_from_archives(txset_slot).expect("should return a map");
 
-		// collector.watch_slot(env_slot);
 		collector.envelopes_map.write().append(env_map);
 
-		// let txset = txsets_map.get(&txset_slot).expect("should return a tx set");
-		// collector.txset_map.write().insert(env_slot, txset.clone());
+		let txset = txsets_map.get(&txset_slot).expect("should return a tx set");
+		collector.txset_map.write().insert(env_slot, txset.clone());
 
 		assert!(collector.envelopes_map.read().contains(&env_slot));
-		//assert!(collector.txset_map.read().contains(&env_slot));
-		// assert!(collector.slot_watchlist.read().contains_key(&env_slot));
+		assert!(collector.txset_map.read().contains(&env_slot));
 
 		collector.remove_data(&env_slot);
+		collector.remove_data(&txset_slot);
```

Comment:
I did not generate new sample files for the `env_slot`, so it makes sense to call `remove_data()` for the `txset_slot`, and check if it was successful.